lib/Web.js

```diff
@@ -31,9 +31,17 @@ function WebBot(configuration) {
             bot.connected = true;
 
             ws.on('message', function incoming(message) {
-
-                var message = JSON.parse(message);
-                controller.ingest(bot, message, ws);
+                try {
+                    var message = JSON.parse(message);
+                    controller.ingest(bot, message, ws);
+                } catch (e) {
+                    console.error(`Message must be a JSON parsable string such as
```

Comment:
Please remove the sample object from the error message.
Replace with this url: https://botkit.ai/docs/readme-web.html#message-objects
Make sure it is properly indented with the rest of the code.
Thanks!